cl/lib/context_processors.py

```diff
@@ -22,24 +22,25 @@ def inject_settings(request):
     }
 
 
+DONATE_URL = "https://donate.free.law/forms/supportflp"
 info_tips = (
     # Other Projects
     'CourtListener is powered by <a href="https://github.com/freelawproject/juriscraper">more than 200 screen scrapers</a> that run every hour of every day.',
     'We are collecting <a href="https://github.com/freelawproject/seal-rookery">all of the court seals in the U.S.</a> You can help contribute seals.',
     'We <a href="https://github.com/freelawproject/judge-pics">have a collection of more than 1,200 judicial portraits</a> and we need volunteers to help collect them all.',
     # History
     'CourtListener was started in 2009 to create alerts for the federal appeals courts. It has since grown into the <a href="%s?referrer=tip">user-supported</a> non-profit Free Law Project.'
-    % reverse("donate"),
+    % DONATE_URL,
     'We host and maintain <a href="https://free.law/2020/03/10/courts-db-a-new-open-database/">a massive database of court identifiers and information</a> we call the Court-DB',
     # Non-profit and fundraising
     'Free Law Project is a 501(c)(3) non-profit that relies on your support to operate. Please <a href="%s?referrer=tip">donate</a> to support this site</a>. <i class="fa fa-heart red"></i>'
-    % reverse("donate"),
+    % DONATE_URL,
     'CourtListener gets more than two million visits per year, but has a lean staff of only a few developers. Please <a href="%s?referrer=tip">donate</a> to support this site. <i class="fa fa-heart red"></i>'
```

Comment:
Time for a stats update by 100×.

```suggestion
    'CourtListener serves more than 200 million requests per year, but has a lean staff of only a handful of developers. Please <a href="%s?referrer=tip">donate</a> to support our work making the judiciary more equitable. <i class="fa fa-heart red"></i>'
```